projects/lib/src/lib/index.ts

```diff
@@ -284,7 +286,7 @@ export const localStorageSync = (config: LocalStorageConfig) => (reducer: any) =
                 nextState,
                 stateKeys,
                 config.storage,
-                config.storageKeySerializer,
+                config.storageKeySerializer as (key: string | number) => string,
```

Comment:
@BBlackwo I actually had changed the type of the function to that, but then realized that it might be a code-breaking change, so changed it back, and did the typecasting in the call instead.

I can change it, if you're okay with the breaking change.

Reply:
Actually I will raise this as a separate issue #187 to confirm what the correct type is and update it everywhere. What you have done is fine for now.